e2e/ia.conf.js

```diff
@@ -2,13 +2,17 @@ module.exports = {
 
   CcdGatewayUrl: process.env.TEST_E2E_URL_GATEWAY || 'https://gateway-ccd.aat.platform.hmcts.net',
   CcdWebUrl: process.env.TEST_E2E_URL_WEB || 'https://www-ccd.aat.platform.hmcts.net',
+  CcdApiUrl: process.env.TEST_E2E_CCD_API_URL || 'http://ccd-data-store-api-aat.service.core-compute-aat.internal',
   UseHeadlessBrowser: process.env.TEST_E2E_HEADLESS !== 'false',
   ProxyUrl: process.env.TEST_E2E_URL_PROXY || 'http://proxyout.reform.hmcts.net:8080',
   RunWithNumberOfBrowsers: process.env.TEST_E2E_NUM_BROWSERS || 1,
   UseProxy: process.env.TEST_E2E_USE_PROXY !== 'false',
   WaitForAngular: process.env.TEST_E2E_WAIT_FOR_ANGULAR !== 'false',
   FailFast: process.env.TEST_E2E_FAIL_FAST !== 'false',
   NightlyTag: process.env.TEST_E2E_ANNOTATION || '--cucumberOpts.tags=@nightly-test',
+  IdamClientSecret: process.env.IDAM_SECRET,
+  IdamApiUrl: process.env.IDAM_API_URL || 'http://idam-api.aat.platform.hmcts.net',
```

Comment:
The README looks out of date but I think there is already an `IA_IDAM_SECRET` `IA_S2S_SECRET` and `IDAM_URL` which is required for the environment setup.  These are probably the same values (as they point to `ia-docker`)

Reply:
as discussed let's update the README to include everything

Reply:
Comment has been superseded by adding .local_env to ia-docker